go/fasthttp/go.mod

```diff
@@ -1,3 +1,3 @@
 module main
```

Comment:
The `config.yml` is missing! ;)

Reply:
sure, the `renovatebot` can't update it, and you are asked as reviewer before (trigger by **github**)